mfr/extensions/zip/templates/viewer.mako

```diff
@@ -1,31 +1,43 @@
-<link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600,300,700" rel="stylesheet" type="text/css">
-<link rel="stylesheet" href="static/css/bootstrap.min.css">
-<link rel="stylesheet" href="static/css/default.css">
-<div style="word-wrap: break-word;" class="mfrViewer">
-    <h1>
-        Zip File:
-    </h1>
-    <p>
-        Download the .zip file to view the contents.
-    </p>
-    ${message}
-    <table class="table table-hover">
-        <thead>
-            <th>File Name</th>
-            <th>Modified</th>
-            <th>Size</th>
-        </thead>
-        <tbody>
-            % for file in zipped_filenames:
-                <tr>
-                    <td>${file['name']}</td>
-                    <td>${file['date']}</td>
-                    <td>${file['size']}<td>
-                </tr>
-            % endfor
-        </tbody>
-    </table>
+<link rel="stylesheet" href="../static/css/bootstrap.min.css">
+<link rel="stylesheet" href="../static/css/default.css">
+<link rel="stylesheet" href="../static/jstree-theme/style.css"/>
```

Comment:
You should be passing in `assets_url` and defining these as
`${base}/static/css......`
and not with `.../static`